be/src/runtime/local_pass_through_buffer.cpp

```diff
@@ -96,4 +95,44 @@ void PassThroughContext::pull_chunks(int sender_id, ChunkUniquePtrVector* chunks
     sender_channel->pull_chunks(chunks, bytes);
 }
 
+void PassThroughChunkBufferManager::open(const TUniqueId& query_id) {
+    VLOG_FILE << "PassThroughChunkBufferManager::open, query_id = " << query_id;
+    {
+        std::unique_lock _l(_mutex);
```

Comment:
local variable may be do not need to start with _

Reply:
sure. will fix that.